### DISCUSSION TITLE
論文發表 其實沒有很難？

### DISCUSSION THREAD

**Discussion by Teddy111802518** [1101-GS4719-C/Python-Community-GS4719-C#2267] [General]
各位 我是一位大三生 在我高中的時候一直覺得發論文很困難 你們覺得呢 感覺要做好多的研究 寫好多報告 想到就覺得可怕 身為一個專題生 教授太恐怖了

**Comment by Teddy111802518** [comment]
但有趣的是 我花了一個暑假拼命做研究 前陣子被指導教授告知 我們要發表國際期刊 這還不是最扯 更扯的是 我是共同第一作者 我其實不是想要跩我只是覺得 我幾個月前還是程式小白 過幾個月我就是分析大師還要發表期刊？怪怪的吧 都不知道自己到底有沒有那個實力 雖然有頭銜申請第一段才能過啦 沒頭銜連面試官的臉都看不到

**Comment by Teddy111802518** [comment]
而且我最近有一個新的軟體開發想法 老師覺得很讚所以決定將此LLM功能導入軟體中 莫名奇妙又多一個publication 還是其實我可能挺有天賦？ 因為我覺得我其實沒有做什麼了不起的事 還是我小看自己了

**Comment by Taoyi-Chiang** [comment]
我覺得還是有一定的門檻，當然很多很水的研討會要發表可能相對簡單很多。
可是大家都會看，就像投稿掠奪性期刊一樣，最後不只是白忙一場，可能還把自己的名譽賠掉。
